package.json

```diff
@@ -26,40 +26,40 @@
     "util.js"
   ],
   "dependencies": {
-    "@stablelib/chacha20poly1305": "^0.10.3",
-    "@stablelib/xchacha20poly1305": "^0.1.1",
+    "@stablelib/chacha20poly1305": "^1.0.0",
+    "@stablelib/xchacha20poly1305": "^1.0.0",
     "base58-universal": "^1.0.0",
     "base64url-universal": "^1.1.0",
     "esm": "^3.2.25",
     "isomorphic-webcrypto": "^2.3.2",
     "tweetnacl": "^1.0.1",
-    "web-streams-polyfill": "^2.0.6"
+    "web-streams-polyfill": "^3.0.2"
```

Comment:
I don't know what this will do ... or what will happen to libs that depend on this. I'm +1 for upgrading, just making a note that we'll need to investigate dependents.

Reply:
@dlongley - I investigated it; I mention it in the CHANGELOG entry here - https://github.com/digitalbazaar/minimal-cipher/pull/32/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR14
short version - we're not affected.

Reply:
@dlongley but I will continue checking on its API as I propagate minimal-cipher downstream.